src/coreclr/jit/emitxarch.cpp

```diff
@@ -11123,6 +11136,11 @@ void emitter::emitDispIns(
         {
             printf("%s, %s, %s", emitRegName(id->idReg1(), attr), emitRegName(id->idReg2(), attr), sstr);
             emitDispAddrMode(id);
+            if (!id->idIsEvexbContext())
+            {
+                break;
+            }
+            emitDispEmbBroadcastCount(id);
```

Comment:
I think I was unclear in my previous suggestion. I was suggesting that here, in emitDispIns, we just have:
```suggestion
            emitDispEmbBroadcastCount(id);
```

And add:
```
            if (!id->idIsEvexbContext())
            {
                return;
            }
```
at the very beginning of `emitDispEmbBroadcastCount()`.

Reply:
Ah, yes, this is more clear. I understood it wrong.